frontend/pages/admin/initiative/new.vue

```diff
@@ -0,0 +1,74 @@
+<script setup lang="ts">
+import { Initiative } from '@/openapi/generated/pacta'
+import { initiativeEditor } from '@/lib/editor'
+
+const prefix = 'admin/initiative/new'
+const router = useRouter()
+const pactaClient = await usePACTA()
+const { loading: { withLoading } } = useModal()
+
+const {
+  editorInitiative,
+  incompleteFields,
+  hasChanges,
+  isIncomplete,
+  initiative,
+} = initiativeEditor({
+  id: '',
+  name: '',
+  affiliation: '',
+  publicDescription: '',
+  internalDescription: '',
+  requiresInvitationToJoin: false,
+  isAcceptingNewMembers: false,
+  isAcceptingNewPortfolios: false,
+  language: Initiative.language.EN,
```

Comment:
Should there be a TODO here to set this based on the user's chosen language?

Reply:
GQ - the requirements specified english as the default.